core/domain/stats_jobs_continuous.py

```diff
@@ -522,29 +522,86 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
-        if InteractionAnswerSummariesMRJobManager._entity_created_before_job_queued( # pylint: disable=line-too-long
-                item):
+        def _encode(item):
+            """Encodes a Python value as UTF-8."""
+            if isinstance(item, basestring):
+                return item.encode('utf-8')
+            elif isinstance(item, int) or isinstance(item, float):
+                return item
+            elif isinstance(item, list):
+                return [_encode(subitem) for subitem in item]
+            elif isinstance(item, dict):
+                return {
+                    key: _encode(value) for key, value in item.iteritems()
```

Comment:
I'm guessing not, but just to check: you don't need to encode the keys? (But if it ain't broken, don't fix it...)